### DISCUSSION TITLE
input attribute disabled only for a part of the template data

### DISCUSSION THREAD

**Discussion by anjanesh** [alpinejs/alpine#3655] [1. Help] [answered]
```
<template x-for="(disk, index) in $store.global_data.disk_details">
	<input type="text" x-model="disk.fields.disk_name" disabled />
</template>
<button
    type="button"
    @click="$store.global_data.disk_details.push({fields: { disk_name: '' }})"
>
    Add New Disk
</button>
```
For new text inputs I need it not disabled.
Only for data for existing fields I need them disabled. 
Is this Possible ?

**Comment by SimoTod** [comment, marked as answer]
You can add a disabled boolean in your disk_details entries, set it to true for the initial disks and false for the ones added dynamically, then bind the disabled attribute based on that value.
